docs/asyncapi.yaml

```diff
@@ -0,0 +1,92 @@
+asyncapi: 3.0.0
+info:
+  title: FCP-FD-COMMS
+  version: 1.0.0
+  description: Receive a communication message and route to appropriate channels
+
+channels:
+  sourceSystem-fcp-fd-comms:
```

Comment:
do we need `sourceSystem-` here?  The channel is consistent for everyone.

Reply:
Was not sure about the naming here. Looking at the example in doc-generator, it seems like convention is sourceSystem-receivingSystem

Reply:
I suspect that's because the doc service didn't want to reference the Payment Hub as the owner of those topics. (as it doesn't care where they come from, just what they look like).

In this case we own the topic so we don't need to reference a source system anywhere